# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/wr/wrench/reftests/filters/reftest.list
# Commit: 79f4180c783b
# Full Hash: 79f4180c783b1e72fccb1e49fb8db086ea12ecca
# Author: Glenn Watson <git@intuitionlibrary.com>
# Date: 2022-05-18 09:47:25
# Regressor Bug: 1749625
# File Overlap Count: 1
# Description:
#   Bug 1749625 - Fix up and re-enable backdrop-filter r=gfx-reviewers,lsalzman
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D146643
# ==============================================================================

diff -r 622b05c84338 -r 79f4180c783b gfx/wr/wrench/reftests/filters/reftest.list
--- a/gfx/wr/wrench/reftests/filters/reftest.list	Wed May 18 03:02:04 2022 +0000
+++ b/gfx/wr/wrench/reftests/filters/reftest.list	Wed May 18 03:29:10 2022 +0000
@@ -61,18 +61,18 @@
 platform(linux,mac) fuzzy(3,3550) == svg-filter-blur-transforms.yaml svg-filter-blur-transforms.png
 platform(linux,mac) == svg-filter-drop-shadow-on-viewport-edge.yaml svg-filter-drop-shadow-on-viewport-edge.png
 fuzzy(1,1) platform(linux,mac) == svg-filter-drop-shadow-perspective.yaml svg-filter-drop-shadow-perspective.png
-#== backdrop-filter-basic.yaml backdrop-filter-basic-ref.yaml
-#platform(linux,mac) == backdrop-filter-perspective.yaml backdrop-filter-perspective.png
-#== backdrop-filter-on-child-surface.yaml backdrop-filter-on-child-surface-ref.yaml
-#== backdrop-filter-clip-mask.yaml backdrop-filter-clip-mask-ref.yaml
-#platform(linux,mac) == backdrop-filter-across-tiles.yaml backdrop-filter-across-tiles.png
-#platform(linux,mac) == backdrop-filter-chain.yaml backdrop-filter-chain.png
-#platform(linux,mac) == backdrop-filter-overlap.yaml backdrop-filter-overlap.png
-#platform(linux,mac) == backdrop-filter-blur-across-tiles.yaml backdrop-filter-blur-across-tiles.png
-#== backdrop-filter-isolation.yaml backdrop-filter-isolation-ref.yaml
+== backdrop-filter-basic.yaml backdrop-filter-basic-ref.yaml
+platform(linux,mac) == backdrop-filter-perspective.yaml backdrop-filter-perspective.png
+== backdrop-filter-on-child-surface.yaml backdrop-filter-on-child-surface-ref.yaml
+== backdrop-filter-clip-mask.yaml backdrop-filter-clip-mask-ref.yaml
+platform(linux,mac) == backdrop-filter-across-tiles.yaml backdrop-filter-across-tiles.png
+platform(linux,mac) == backdrop-filter-chain.yaml backdrop-filter-chain.png
+platform(linux,mac) == backdrop-filter-overlap.yaml backdrop-filter-overlap.png
+platform(linux,mac) == backdrop-filter-blur-across-tiles.yaml backdrop-filter-blur-across-tiles.png
+== fuzzy(1,4900) backdrop-filter-isolation.yaml backdrop-filter-isolation-ref.yaml
 #platform(linux,mac) == backdrop-filter-drop-shadow.yaml backdrop-filter-drop-shadow.png
-#== backdrop-filter-blur-edge-mode.yaml backdrop-filter-blur-edge-mode-ref.yaml
-#== backdrop-filter-half-opacity.yaml backdrop-filter-half-opacity-ref.yaml
+== backdrop-filter-blur-edge-mode.yaml backdrop-filter-blur-edge-mode-ref.yaml
+== backdrop-filter-half-opacity.yaml backdrop-filter-half-opacity-ref.yaml
 platform(linux,max) == svg-filter-offset.yaml svg-filter-offset-ref.yaml
 skip_on(android,device) == fuzzy(1,100) svg-filter-composite.yaml svg-filter-composite-ref.yaml
 skip_on(android,device) == filter-mix-blend-scaling.yaml filter-mix-blend-scaling-ref.yaml